### DISCUSSION TITLE
BAC0.core.devices.local.models

### DISCUSSION THREAD

**Discussion by rubada** [ChristianTremblay/BAC0#510] [General]
Hi I want to ask why did you remove the BAC0.core.devices.local.models module from this new version

**Comment by ChristianTremblay** [comment]
Because I switched to another way of creating local object using what I called the factory. It was more flexible and fixed some troubles I had with the initial models.

Have a look there, you will see how I build local objects

https://github.com/ChristianTremblay/BAC0/blob/main/tests/manual_test_create_device.py

**Reply by rubada** [reply]
Thank you
